xclim/indices/_multivariate.py

```diff
@@ -239,6 +239,8 @@ def daily_temperature_range(
       Maximum daily temperature values [℃] or [K]
     freq : str
       Resampling frequency; Defaults to "YS".
+    op : str {'min', 'max', 'mean', 'std', 'var', 'count', 'sum', 'argmax', 'argmin'} or func
```

Comment:
Does this mean that you can send a callable method like `np.mean`?

Reply:
yes you can send in as func.... not sure if my typing is ok for the call signature here.  Copied what was is indices.generic

Reply:
All good. Just wanted to make sure that this is an allowed argument. Helps me better implement typing on the mypy branch.